code/modules/mob/living/carbon/human/species/station/station_special_abilities_vr.dm

```diff
@@ -80,55 +77,56 @@
 	set name = "Hatch"
 	set category = "Abilities"
 
-	if(world.time < last_special)
-		return
-
-	last_special = world.time + 50 //To prevent button spam.
-
 	var/confirm = alert(usr, "Are you sure you want to hatch right now? This will be very obvious to anyone in view.", "Confirm Regeneration", "Yes", "No")
-	if(confirm == "Yes")
-		if(stat == DEAD) //Uh oh, you died!
-			if(hasnutriment()) //Let's hope you have nutriment in you.... If not
-				if(src) //Runtime prevention.
-					chimera_hatch()
-					visible_message("<span class='danger'><p><font size=4>The lifeless husk of [src] bursts open, revealing a new, intact copy in the pool of viscera.</font></p></span>") //Bloody hell...
-					brainloss += 10 //Reviving from dead means you take a lil' brainloss on top of whatever was healed in the revive.
-					return
-				else
-					return //Runtime prevention
-			else //don't have nutriment to hatch! Or you somehow died in between completing your revive and hitting hatch.
-				to_chat(src, "Your body was unable to regenerate, what few living cells remain require additional nutrients to complete the process.")
-				reviving = FALSE // so they can try again when they're given a kickstart
+	if(confirm == "Yes" && reviving)
+
+		//Dead when hatching
+		if(stat == DEAD)
+			//Check again for nutriment (necessary?)
+			if(hasnutriment())
+				chimera_hatch()
+				visible_message("<span class='danger'><p><font size=4>The lifeless husk of [src] bursts open, revealing a new, intact copy in the pool of viscera.</font></p></span>") //Bloody hell...
+				brainloss += 10 //Reviving from dead means you take a lil' brainloss on top of whatever was healed in the revive.
```

Comment:
Apparently just setting the brainloss var doesn't work, just VV'd after reviving from dead and it was 0. Might need to call adjustBrainLoss() instead.